src/main/webapp/app/modules/provider/all-records.tsx

```diff
@@ -383,8 +394,13 @@ export class AllRecords extends React.Component<IAllRecordsProps, IAllRecordsSta
         </MediaQuery>
         <MediaQuery minDeviceWidth={DESKTOP_WIDTH_BREAKPOINT}>
           <Row className="mb-4 mx-3">
-            <Col md={isRecordHighlighted || filterOpened ? 8 : 12} className="pb-2 pl-0 pr-1 map-view">
+            <Col md={isRecordHighlighted || filterOpened ? 8 : 12} className="pb-2 pl-0 pr-1 map-view position-relative">
               <Map {...mapProps} containerElement={<div style={{ height: '400px' }} />} />
+              <div className="position-absolute" style={{ right: '65px', bottom: '32px' }}>
```

Comment:
do we need position absolute and inline styles here? maybe let's use some constant or existing class